e2e/nomostest/portforwarder/port_forwarder.go

```diff
@@ -31,6 +31,16 @@ import (
 	"kpt.dev/configsync/pkg/kinds"
 )
 
+var errTerminal = errors.New("terminal PortForwarder error")
```

Comment:
I initially was going to reuse that one, but I decided to scope this error to be private to this package so that consumers don't need to worry about this error. All of the checking for terminal errors is done within the package

Reply:
The only reason to expose it, I think, would be if someone wanted to use it in the callback, right?
Or are callback errors always terminal?

Reply:
The callback does not return an error. Errors within the callback call `Fatal`

Reply:
We should probably make them return errors in a follow up PR, and define the expected behavior when the callback errors..